src/View/mainpage/PatientBar.py

```diff
@@ -21,23 +47,19 @@ def __init__(self):
         self.icon.setMaximumWidth(30)
         self.icon.setGeometry(QtCore.QRect(10, 0, 0, 0))
         self.icon.setText("")
-        self.icon.setPixmap(QtGui.QPixmap(":/images/Icon/patient.png"))
+        self.icon.setPixmap(QtGui.QPixmap("src/Icon/patient.png"))
 
         # Create patient name
-        self.name_widget = QtWidgets.QWidget()
-        self.create_patient_name()
+        self.name_widget = create_info_widget("Name", self.basic_info['name'])
```

Comment:
You seem to have lost the Geometry information.
Is that OK?
I think you want to pass in the Geometry, at least optionally (named parameter).